config/base/dex/numaflow-dex-server-configmap.yaml

```diff
@@ -4,30 +4,29 @@ metadata:
   name: numaflow-dex-server
 data:
   config.yaml: |
-    issuer: <HOSTNAME>/dex
+    issuer: https://localhost:8443/dex
     storage:
       type: memory
     web:
       http: 0.0.0.0:5556
     staticClients:
       - id: numaflow-server-app
         redirectURIs: 
-          - <HOSTNAME>/<base_href>/login
+          - https://localhost:8443/login
         name: 'Numaflow Server App'
         public: true
     connectors:
     - type: github
-      # https://dexidp.io/docs/connectors/github/
       id: github
       name: GitHub
       config:
         clientID: $GITHUB_CLIENT_ID
         clientSecret: $GITHUB_CLIENT_SECRET
-        redirectURI: <HOSTNAME>/dex/callback
+        redirectURI: https://localhost:8443/dex/callback
```

Comment:
Same as my `Authorization callback URL` of https://github.com/settings/applications/2368167